main/src/cgeo/geocaching/settings/OCMaxDistancePreference.java

```diff
@@ -0,0 +1,112 @@
+package cgeo.geocaching.settings;
+
+import cgeo.geocaching.R;
+import cgeo.geocaching.location.IConversion;
+
+import android.content.Context;
+import android.preference.Preference;
+import android.util.AttributeSet;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.SeekBar;
+import android.widget.SeekBar.OnSeekBarChangeListener;
+import android.widget.TextView;
+
+import java.util.Locale;
+
+import butterknife.ButterKnife;
+
+public class OCMaxDistancePreference extends Preference {
```

Comment:
I think that methods\classes in common `settings` folder should not be tightened to particular service